spec/features/switching_policies_spec.rb

```diff
@@ -1,17 +1,29 @@
 require "rails_helper"
 
 RSpec.feature "A user can switch policies" do
-  it "a user can switch to maths and physics after starting a student loan claim" do
-    start_student_loans_claim
-    visit new_claim_path(MathsAndPhysics.routing_name)
+  context "after starting a student loan claim" do
```

Comment:
```suggestion
  context "when trying to start a maths and physics claim, with a student loans claim already in progress" do
```

(and then `a user can switch to maths and physics` for the `scenario`)